benchmark/common.js

```diff
@@ -1,205 +1,207 @@
 'use strict';
-var assert = require('assert');
-var fs = require('fs');
-var path = require('path');
-var child_process = require('child_process');
-
-var outputFormat = process.env.OUTPUT_FORMAT ||
-                   (+process.env.NODE_BENCH_SILENT ? 'silent' : false) ||
-                   'default';
-
-// verify outputFormat
-if (['default', 'csv', 'silent'].indexOf(outputFormat) == -1) {
-  throw new Error('OUTPUT_FORMAT set to invalid value');
-}
 
-exports.PORT = process.env.PORT || 12346;
+const child_process = require('child_process');
 
-function hasWrk() {
-  var result = child_process.spawnSync('wrk', ['-h']);
-  if (result.error && result.error.code === 'ENOENT') {
-    console.error('Couldn\'t locate `wrk` which is needed for running ' +
-      'benchmarks. Check benchmark/README.md for further instructions.');
-    process.exit(-1);
-  }
+// parse configuarion arguments
+const configArgs = {};
+for (const arg of process.argv.slice(2)) {
+  const match = arg.split('=');
+  const isNumber = isFinite(match[1]) && match[1] !== '';
+  configArgs[match[0]] = isNumber ? parseFloat(match[1]) : match[1];
```

Comment:
not sure how much we care about this, but doing the check so simply arguments like `--foo=42=bar` will pass. May consider switching it to `arg.split(/^([^=]+)=/)` or something similar.